### PR TITLE
Refactor: remove dead code

### PR DESCRIPTION
**Pull request by shubham0804**
## Type of Change

- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement
- [x] Refactoring
- [ ] Dependency updates
- [ ] Documentation
- [ ] CI/CD

## Description
Removed dead code. This PR depends on #133 . Kindly merge/review that before this one

The package `react-qrcode-logo` has been removed because the only component where it was being used was dead code.

## Motivation and Context
Issue #108 

## How did you test it?
Running `npm run re:build` did not throw any errors

## Checklist

- [x] I ran `npm run re:build`
- [x] I reviewed submitted code
- [ ] I added unit tests for my changes where possible

### CODE REVIEW

**Comment by PritishBudhiraja** [comment]
Please remove all the warnings from the console.

**Comment by shubham0804** [comment]
@PritishBudhiraja I have removed the warnings that were appearing when running `npm run re:build`. I'm unable to change the label of the pr

**Comment by PritishBudhiraja** [comment]
Hey @shubham0804 ,

The code is not getting compiled. Can you please take a pull and check which places issues are there?
We can close this PR ASAP whenever you can fix the errors.

Thanks.

**Comment by shubham0804** [comment]
Hi @PritishBudhiraja The code should compile now. Thanks

File: src/screens/HyperSwitch/Utils/CurrencyUtils.res [hunk 1]
```diff
@@ -310,358 +310,3 @@ let getCurrencyCodeStringFromVariant = currencyCode => {
   | USD => "USD"
   }
 }
-
-let getCurrencyCodeFromString: string => currencyCode = currencyCode => {
```
Comment (PritishBudhiraja) [inline src/screens/HyperSwitch/Utils/CurrencyUtils.res:314]: Can you please revert this file as it might be needed.

File: src/components/DateRangePicker.res [hunk 2]
```diff
@@ -6,15 +6,10 @@ let defaultCellHighlighter = (_): Calendar.highlighter => {
...
-  value: string,
-  setValue: (string => string) => unit,
-  isoStringToCustomTimeZone,
-) => {
+let useErroryValueResetter = (value: string, setValue: (string => string) => unit) => {
   React.useEffect0(() => {
-    let isErroryTimeValue = value => {
+    let isErroryTimeValue = _ => {
       try {
-        let _checkEnd = value->isoStringToCustomTimeZone
```
[the remark this answers is not in the document]
Comment (shubham0804) [reply src/components/DateRangePicker.res:17]: Hi @Riddhiagrawal001 This variable did not seem to be used anywhere. Let me know if I'm mistaken

File: src/screens/HyperSwitch/Utils/CurrencyUtils.res (hunk 1 above)
Comment (shubham0804) [reply src/screens/HyperSwitch/Utils/CurrencyUtils.res]: Done

File: src/screens/HyperSwitch/Utils/CountryUtils.res [hunk 3]
```diff
@@ -751,259 +751,6 @@ type countiesCode =
   | ZM
   | ZW
 
-let getCountryFromCountryCode = code => {
```
Comment (PritishBudhiraja) [inline src/screens/HyperSwitch/Utils/CountryUtils.res:754]: Can you please revert these changes of CountryUtils as they might be getting used.

Comment (shubham0804) [reply src/screens/HyperSwitch/Utils/CountryUtils.res]: Done

**Comment by shubham0804** [comment]
> Shubham is this PR still active - #133?

Yes, #133 contains dead code removal as well. But merging either of #133 or this pr will probably result in conflicts in the other. Will fix the conflicts when either one is merged

File: src/screens/HyperSwitch/UserManagement/ShowUserData.res [hunk 4]
```diff
@@ -6,23 +6,9 @@ external typeConversion: array<Js.Nullable.t<UserRoleEntity.userTableTypes>> =>
 
 module UserHeading = {
   @react.component
-  let make = (~infoValue: UserRoleEntity.userTableTypes, ~userId) => {
```
Comment (PritishBudhiraja) [inline src/screens/HyperSwitch/UserManagement/ShowUserData.res:9]: Hey @shubham0804 can you revert this file also as this in WIP. So let it be the warnings in this.

**Comment by PritishBudhiraja** [comment]
Everything is fine to me. Just revert one [ShowUserData.res](https://github.com/juspay/hyperswitch-control-center/pull/138/files/c0bf13796c5df031accca4a8d46fbdee0a90ff99#diff-b7132dc224b7b29d47131ab6b1bd0b04b52a367e7ef4c1effb2ff33de8ecf5e9) file. Then we can approve and merge the PR.

File: src/screens/HyperSwitch/UserManagement/ShowUserData.res (hunk 4 above)
Comment (shubham0804) [reply src/screens/HyperSwitch/UserManagement/ShowUserData.res]: Hi @PritishBudhiraja. Done

**Comment by shubham0804** [comment]
> Everything is fine to me. Just revert one [ShowUserData.res](https://github.com/juspay/hyperswitch-control-center/pull/138/files/c0bf13796c5df031accca4a8d46fbdee0a90ff99#diff-b7132dc224b7b29d47131ab6b1bd0b04b52a367e7ef4c1effb2ff33de8ecf5e9) file. Then we can approve and merge the PR.

Reverted

File: src/components/DateRangePicker.res (hunk 2 above)
[the remark this answers is not in the document]
Comment (PritishBudhiraja) [reply src/components/DateRangePicker.res:17]: It's okay shubham we can close this. It's not getting used.

**Review by Riddhiagrawal001 — APPROVED** [review]

**Review by PritishBudhiraja — APPROVED** [review]

**Review by JeevaRamu0104 — APPROVED** [review]

**Comment by PritishBudhiraja** [comment]
Hey @shubham0804 ,
Thanks a bunch for all your contributions! We've got a little something for you to show our appreciation.
Just take a moment to fill out this [form](https://docs.google.com/forms/d/e/1FAIpQLSfX0vGppUgtA1qQ6I0rCTfJEab58LV5joMvwY9g-Dd0ySIULA/viewform), and get ready for some awesome swag coming your way.
Thanks!

**Comment by shubham0804** [comment]
> Hey @shubham0804 , Thanks a bunch for all your contributions! We've got a little something for you to show our appreciation. Just take a moment to fill out this [form](https://docs.google.com/forms/d/e/1FAIpQLSfX0vGppUgtA1qQ6I0rCTfJEab58LV5joMvwY9g-Dd0ySIULA/viewform), and get ready for some awesome swag coming your way. Thanks!

Hi @PritishBudhiraja Thanks. Much appreciated! Would love to work on other issues/features. Please do create Issues and feel free to assign them to me. I can also get on a call for coordination if required.
